events/code_review_events/workflow.py

```diff
@@ -53,18 +58,29 @@ def __init__(
 
         # Setup Taskcluster community hooks for risk analysis
         if community_config is not None:
-            self.community_hooks = Hooks(
-                {
-                    "rootUrl": "https://community-tc.services.mozilla.com",
-                    "credentials": {
-                        "clientId": community_config["client_id"],
-                        "accessToken": community_config["access_token"],
-                    },
-                }
-            )
+            self.community_tc = {
```

Comment:
It would be better to have a second instance of TaskclusterConfig here

Reply:
I wanted to do that in a follow-up, once we have the taskcluster.helper library in the official taskcluster.

Reply:
I filed https://github.com/mozilla/code-review/issues/267.